cmake/Macros.cmake

```diff
@@ -3,23 +3,29 @@ include(CMakeParseArguments)
 # add a new target which is a SFML library
 # ex: sfml_add_library(sfml-graphics
 #                      SOURCES sprite.cpp image.cpp ...
-#                      DEPENDS sfml-window sfml-system
-#                      EXTERNAL_LIBS opengl freetype ...)
+#                      [STATIC_ONLY]) # Always create a static library and ignore BUILD_SHARED_LIBS
```

Comment:
I'd probably just call it "STATIC", mirroring CMake's keyword for `add_library()`.

Reply:
Indeed, changing